src/classes/OrgConfig.cls

```diff
@@ -46,7 +46,7 @@ public inherited sharing class OrgConfig {
     /*******************************************************************************************************
     * @description map from State name (UPPER CASE) to State Code
     */
-    private Map<String, String> validStatesByLabel = new Map<String, String>();
+    private static Map<String, String> validStatesByLabel = new Map<String, String>();
```

Comment:
I'm curious why the switch to static, and why not all methods use static.

Reply:
Instance methods for mocking, static fields since not all instances of this class need their own version of this map.